app/_src/gateway/admin-api/workspaces/reference.md

```diff
@@ -295,6 +295,13 @@ workspace itself can be.
 HTTP 204 No Content
 ```
 
+
+Recursively delete a workspace. Normally, deleting a workspace requires its entities to be deleted first. This option allows you to recursively delete a workspace without manually deleting its entities first.
+
+```
+DELETE /workspaces/{name or id}?recursive=true
+```
+
```

Comment:
```suggestion

### Recursively delete a workspace

Normally, deleting a workspace requires its entities to be deleted first. This option allows you to recursively delete a workspace without manually deleting its entities first.

<div class="endpoint delete">/workspaces/{name or id}?recursive=true</div>

```

Reply:
How about:


### Recursively delete a workspace

Deleting a workspace requires first deleting its entities. This option allows you to delete a workspace without manually deleting its entities first, as it handles the entity deletion before the workspace is deleted.

<div class="endpoint delete">/workspaces/{name or id}?recursive=true</div>